### PR TITLE
fix typo for sourceActorSystem and mistake package name

### PR DESCRIPTION
**Pull request by laglangyue**
as mentioned in the title.

### CODE REVIEW

File: testkit/src/main/scala/org/apache/pekko/stream/connectors/testkit/CapturingAppender.scala [hunk 1]
```diff
@@ -56,8 +56,8 @@ import org.slf4j.LoggerFactory
  * the captured logging events are flushed to the appenders defined for the
  * org.apache.pekko.actor.testkit.typed.internal.CapturingAppenderDelegate logger.
  *
- * The flushing on test failure is handled by [[pekko.actor.testkit.typed.scaladsl.LogCapturing]]
- * for ScalaTest and [[pekko.actor.testkit.typed.javadsl.LogCapturing]] for JUnit.
+ * The flushing on test failure is handled by [[org.apache.pekko.actor.testkit.typed.javadsl.LogCapturing]]
+ * for ScalaTest and [[org.apache.pekko.actor.testkit.typed.scaladsl.LogCapturing]] for JUnit.
```
Comment (pjfanning) [inline testkit/src/main/scala/org/apache/pekko/stream/connectors/testkit/CapturingAppender.scala:60]: this change is incorrect

**Review by pjfanning — CHANGES_REQUESTED** [review]

File: testkit/src/main/scala/org/apache/pekko/stream/connectors/testkit/CapturingAppender.scala (hunk 1 above)
Comment (Roiocam) [reply testkit/src/main/scala/org/apache/pekko/stream/connectors/testkit/CapturingAppender.scala:60]: +1

Comment (laglangyue) [reply testkit/src/main/scala/org/apache/pekko/stream/connectors/testkit/CapturingAppender.scala]: done

**Review by raboof — APPROVED** [review]

**Review by pjfanning — APPROVED** [review]
lgtm - thanks

**Pull request merged**